packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -1060,6 +1060,31 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
   ///  * [ModalBarrier], the widget that implements this feature.
   String get barrierLabel;
 
+  /// The tween that is used for animating the modal barrier in and out.
+  ///
+  /// The modal barrier is the scrim that is rendered behind each route, which
+  /// generally prevents the user from interacting with the route below the
+  /// current route, and normally partially obscures such routes.
+  ///
+  /// For example, when a dialog is on the screen, the page below the dialog is
+  /// usually darkened by the modal barrier.
+  ///
+  /// While the route is animating into position, the color is animated from
+  /// transparent to the specified color.
+  ///
+  /// If this getter would ever start returning a different color,
```

Comment:
This sentence is odd, the getter doesn't return a color?

Reply:
Sorry, this was a remnant of copying the docs from the rest of the `barrier__` properties. I notice that it's the same way for the non-`barrierColor` properties and updated them in those places since those didn't make sense either.